sympy/ntheory/__init__.py

```diff
@@ -14,7 +14,7 @@
 from .residue_ntheory import is_primitive_root, is_quad_residue, \
     legendre_symbol, jacobi_symbol, n_order, sqrt_mod, quadratic_residues, \
     primitive_root, nthroot_mod, is_nthpow_residue, sqrt_mod_iter, mobius, \
-    discrete_log
+    discrete_log, collatz
```

Comment:
This isn't really related to residue number theory, but then it doesn't seem to fit anywhere.

Reply:
It might go in with Fibonacci and the like since it is a sequence. crypto might be another possibility since (as I mentioned in the link) there are cryptographic properties to the sequence.

Reply:
I was also not sure that where it will get fit. Fibonacci is a series and it's always start with same sequence. This is not the case with collatz. I don't think that Fibonacci and Collatz should go together.